edi_af_appointment/controllers/main.py

```diff
@@ -6,19 +6,20 @@
 import pytz
 
 import logging
+
 _logger = logging.getLogger(__name__)
 
 # DUPLICATES IN calendar_af
-LOCAL_TZ = 'Europe/Stockholm'
+LOCAL_TZ = "Europe/Stockholm"
 BASE_DURATION = 30.0
 # RESERVED_TIMEOUT is the default time before a reservation times out.
 RESERVED_TIMEOUT = 300.0
 
-class AppointmentController(http.Controller):
 
+class AppointmentController(http.Controller):
```

Comment:
This function is wrongly implemented as it does return either False or None
return int(string)